app.rb

```diff
@@ -14,6 +14,9 @@ def run
       exit if operation == 'q'
 
       @options.choose_option operation
+      @options.save_books_to_file
+      @options.save_people_to_file
+      @options.save_rental_data
```

Comment:
- [ ] Great job preserving the data! However, the `name` is being stored/loaded as `true`:

Look at person #7:

![Screen Shot 2022-07-30 at 10 24 31 AM](https://user-images.githubusercontent.com/83717285/181923499-dfe0c439-d872-42d9-ac55-83aab602efd0.png)

It would be great to fix this bug and display the `name`.